gamearea.py

```diff
@@ -650,7 +650,7 @@ def cb():
         self.add_cats()
         self.playing = True
 
-        self.start_timeout(15, cb)
+        self.start_timeout(15 + 2 * (self.level-2), cb)
```

Comment:
What's the reasoning here?

Reply:
Corrected the level initialization see this one first https://github.com/sugarlabs/classify-cats/pull/12.
Corrected the logic here also as 15 + 2*(self.level-1);
logic is base time is 15 sec:
then for level 1 = 15+2*(1-1) = 15;
then for level 2 = 15+2*(2-1) = 17; and so on.

Reply:
If a problem occurred here, why open another PR to fix it?

Reply:
I got confused as previously I though game was starting at level one thats why I used - 2 as control but then made another branch to test and see whats exactly happening it is related to how level.json file is being accessed and also both of them are unrelated one is bug other is feature though could have been made in one.